mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
I am lost, what is this copy https://github.com/radical/mono/blob/3d7b40dd0c46ff8614b2cc05897d1665278e74a9/mcs/packages/Makefile#L43 about ?